custom_components/nws_alerts/__init__.py

```diff
@@ -280,12 +280,15 @@ async def async_get_alerts(zone_id: str = "", gps_loc: str = "") -> dict:
             tmp_dict["Severity"] = alert["properties"]["severity"]
             tmp_dict["Certainty"] = alert["properties"]["certainty"]
             tmp_dict["Sent"] = alert["properties"]["sent"]
+            tmp_dict["Effective"] = alert["properties"]["effective"]            
             tmp_dict["Onset"] = alert["properties"]["onset"]
             tmp_dict["Expires"] = alert["properties"]["expires"]
             tmp_dict["Ends"] = alert["properties"]["ends"]
             tmp_dict["AreasAffected"] = alert["properties"]["areaDesc"]
             tmp_dict["Description"] = alert["properties"]["description"]
-            tmp_dict["Instruction"] = alert["properties"]["instruction"]
```

Comment:
This property got deleted.   Was it intentional?

Reply:
This was a mistake. I've been out of pocket for a while.